### ISSUE TITLE
WCF Data Service UI doesn't show up in Visual Studio 2017

### ISSUE THREAD

**Issue by AlanWong-MS** [OData/odata.net#1079]
WCF Data Service UI doesn't show up in Visual Studio 2017.

### Assemblies affected
ODL 5.6.4
Microsoft.Data.Services.Client 5.6.4
Microsoft.Data.Services 5.6.4

### Reproduce steps
"In a "ASP.NET Web Application" project we right click in Solution Explorer to get a menu where we have a choice to Add "WCF Data Service" item to the project. This menu item is not available and I'm not aware if there is any other way of creating a WCF data service which I wish to consume from UWP Win10 app. Other than the regular right click menu on Solution Explorer there is no screen shot."

### Expected result
There should be a menu context item to add "WCF Data Service" item to the project upon installation.

### Actual result
The menu context item is not there.

### Additional detail
Original thread: https://developercommunity.visualstudio.com/content/problem/25983/wcf-data-service-template-missing-in-vs2017.html

**Comment by hjekman** [comment]
I'd say this applies to installing VS 2017 in general and not specifically OData or UWP Win 10 apps, wonder why Microsoft has chosen to hide it here in git and closed/moved it off visualstudio dev com :/

**Comment by philanderson888** [comment]
I'm a teacher and trying to follow the instructions for creating and using a WCF Data Service.  None of the fixes provided in the above thread https://developercommunity.visualstudio.com/content/problem/25983/wcf-data-service-template-missing-in-vs2017.html seem to work for me so I'm puzzled as to what is the solution.

**Comment by Dosihris** [comment]
So are there any news on that issue? Visual Studio 2019 still hasn't WCF Data Service Template!

**Comment by kathleenwest** [comment]
Yes, we should have access to the existing templates like WCF Data Service for backwards compatibility.

**Comment by gathogojr** [comment]
Options to add "WCF Data Service" to project no longer supported
